shared/versions/stack/master.asciidoc

```diff
@@ -1,19 +1,19 @@
-:version:                8.7.0
+:version:                8.9.0
 ////
 bare_version never includes -alpha or -beta
 ////
-:bare_version:           8.7.0
-:logstash_version:       8.7.0
-:elasticsearch_version:  8.7.0
-:kibana_version:         8.7.0
-:apm_server_version:     8.7.0
+:bare_version:           8.9.0
+:logstash_version:       8.9.0
+:elasticsearch_version:  8.9.0
+:kibana_version:         8.9.0
+:apm_server_version:     8.9.0
 :branch:                 master
-:minor-version:          8.7
+:minor-version:          8.9
 :major-version:          8.x
 :prev-major-version:     7.x
 :prev-major-last:        7.17
 :major-version-only:     8
-:ecs_version:            8.8
+:ecs_version:            8.9
```

Comment:
> :ecs_version:            8.9

It looks like this will need to be either `8.8` or `master` for now, since there's no 8.9 yet in https://www.elastic.co/guide/en/ecs/8.8/ecs-field-reference.html

Reply:
```suggestion
:ecs_version:            master
```